### DISCUSSION TITLE
[Idea]Release notes URL

### DISCUSSION THREAD

**Discussion by sebma** [glpi-project/glpi-agent#771] [Ideas]
### Your idea

Hi, Can you please add the release notes URL on the [Releases](../releases) page ?

**Comment by g-bougard** [comment]
Hi @sebma 

I just added it for v1.11. And we will try to remind to do it for later releases.

**Comment by sebma** [comment]
@g-bougard Thanks.

**Comment by erique-souza** [comment]
This is really good, the return of path notes on the releases page, I missed it in the last few releases, it makes it easier to find recent changes
